bin/bump.sh

```diff
@@ -47,7 +49,14 @@ checkChangesSinceCurrentVersion() {
   local currentVersion
   local changes
   currentVersion=$(npm ls --depth=-1 2>/dev/null | head -1 | cut -f 1 -d " " | sed 's/.*@//')
-  changes=$(git --no-pager log v"$currentVersion".. --oneline --no-merges --reverse)
+  changes=$(git --no-pager log v"$currentVersion".. --oneline --reverse)
+
+  if [ "$VERSION" == "patch" ] || [ "$VERSION" == "minor" ]; then
+    if echo "$changes" | grep --quiet --ignore-case breaking; then
```

Comment:
I wonder if there will be a case where the commit message contains "breaking" even when it's not a breaking change, such as a case where it is mentioning previous breaking changes. It might be better to just enforce starting with the exact tag "BREAKING".